store/postgres/src/relational_queries.rs

```diff
@@ -362,6 +367,14 @@ impl FromColumnValue for r::Value {
         }
     }
 
+    fn from_timestamp(i: &str) -> Result<Self, StoreError> {
+        scalar::Timestamp::from_rfc3339(i)
+            .map(|v| r::Value::Int(v.as_millis_since_epoch()))
```

Comment:
If we had a `r::Value::Timestamp` we could avoid this conversion. And isn't that going to run into integer overflow at some point>

Reply:
Adding